### PR TITLE
TestsProjects

### PR DESCRIPTION
**Pull request by denis-kriv**
Added test projects for each service

### CODE REVIEW

**Review by KostinP — APPROVED** [review]

File: DigitalOffice/DigitalOffice.sln [hunk 1]
```diff
@@ -15,6 +15,20 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FileService", "Services\Fil
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TimeManagementService", "Services\TimeManagementService\TimeManagementService.csproj", "{BFEC18E6-CA3F-42AC-AB32-AEBA70AB9603}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{726AAF26-0B24-4525-99A3-955F0E354C71}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CheckRightsServiceTests", "Tests\CheckRightsServiceTests\CheckRightsServiceTests.csproj", "{B1995E22-8ED7-4C89-97CE-B062533E5BE8}"
```
Comment (DarkFelas) [inline DigitalOffice/DigitalOffice.sln]: All projects have wrong name, read task again.

**Review by DarkFelas — CHANGES_REQUESTED** [review]

File: DigitalOffice/DigitalOffice.sln (hunk 1 above)
Comment (denis-kriv) [reply DigitalOffice/DigitalOffice.sln]: fixed

**Review by lalagvana — APPROVED** [review]

File: DigitalOffice/Tests/CheckRightsServiceTests/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace CheckRightsServiceTests
```
Comment (DarkFelas) [inline DigitalOffice/Tests/CheckRightsServiceTests/UnitTest1.cs]: Fix namespaces for all projects

**Review by DarkFelas — CHANGES_REQUESTED** [review]

File: DigitalOffice/Tests/CheckRightsServiceTests/UnitTest1.cs (hunk 2 above)
Comment (denis-kriv) [reply DigitalOffice/Tests/CheckRightsServiceTests/UnitTest1.cs]: fixed

**Review by DarkFelas — APPROVED** [review]

**Review by Inzilbeth — APPROVED** [review]

**Review by mekk1t — APPROVED** [review]

**Pull request merged**